ensime_shared/ensime.py

```diff
@@ -300,3 +362,13 @@ def fun_en_complete_func(self, client, findstart_and_base, base=None):
             # Invoked by vim
             findstart = findstart_and_base
         return client.complete_func(findstart, base)
+
+    def watchdog_client_server_connection(self, client):
```

Comment:
This isn't working (for me), but don't worry, I will take a look as part of #350